### ISSUE TITLE
Dice Score Becomes `nan` After Epoch 130 in xLSTM-uNet 3d_fullres Model Training

### ISSUE THREAD

**Issue by matteodelucchi** [tianrun-chen/xLSTM-UNet-PyTorch#13]
I am experiencing an issue with the xLSTM-uNet model, where the Dice score becomes `nan` after epoch 130. Up until this point, the Dice score has increased, and the losses have decreased as expected. However, at epoch 130, the validation Dice score suddenly drops to `nan`.
![progress](https://github.com/user-attachments/assets/b31be333-5085-40b7-94c8-7ff33b7f593c)
Possible Causes: I suspect issues with exploding gradients or divisions by zero but not sure... 
- Is there a way to extract the gradients?
- What are suggested regularization techniques?
I would appreciate any guidance or suggestions on how to resolve this issue. Thank you!
